actionlist/app/actions/types.js

```diff
@@ -1,6 +1,8 @@
-export const TOGGLE_DRAWER = 'toggle_drawer';
+export const OPEN_DRAWER = 'open_drawer';
+export const CLOSE_DRAWER = 'close_drawer';
 export const FILTER_ACTION_LIST = 'filter_action_list';
 export const SORT_ACTION_LIST = 'sort_action_list';
 export const SELECT_DROPDOWN_OPTION = 'select_dropdown_option';
 export const RESET_FILTERS = 'reset_filters';
+export const DATE_FILTER_CHANGE = 'date_filter_change';
```

Comment:
So would we have a new namespace "filters" within "actions" and we will have these types imported and assigned to actions there?
I am not really clear about this change